cmd/juju/application/trust.go

```diff
@@ -19,19 +20,30 @@ const (
 	trustSummary = `Sets the trust status of a deployed application to true.`
 	trustDetails = `Sets the trust configuration value to true.
 
+On k8s models, the trust operation currently grants the charm full access to the cluster.
+Until the permissions model refined to grant more granular role based access, the use of
+'--scope=cluster' is required to confirm this choice.
+
 Examples:
     juju trust media-wiki
+    juju trust metallb --scope=cluster
 
 See also:
     config
+`
+	clusterScopeError = `'juju trust' currently grants full access to the cluster itself.
+Set the scope to 'cluster' using '--scope=cluster' to confirm this choice.
```

Comment:
I have some concern that this functionally puts us into a forced cli with '--scope', and we haven't even designed how we want categories to work. And what would the syntax be if there are multiple things that could be trusted.

Reply:
Agree, but the clear direction we were given is that this is what was wanted in the dot zero release.